components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -127,7 +127,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- ================================================================== -->
 <!-- FATES default parameter file                                       -->
 <!-- ================================================================== -->
-<fates_paramfile >lnd/clm2/paramdata/fates_params_api25.5.0_12pft_c230710.nc</fates_paramfile>
+<fates_paramfile >lnd/clm2/paramdata/fates_params_api.32.0.0_12pft_c231215.nc</fates_paramfile>
```

Comment:
This file is not available on https://web.lcrc.anl.gov/public/e3sm/inputdata/lnd/clm2/paramdata/

Reply:
Thanks @bishtgautam I'll email you the file.

Reply:
The file is on the other server though.  https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/paramdata/

Reply:
Oh, FATES also contributes code to CTSM, so it is on NCAR's server. In that case, we don't need to put on the E3SM server.

Reply:
Oh nice, I didn't realize that!  Thanks @peterdschwartz